### PR TITLE
clojurescript 1.10.844

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/clojurescript.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Clojurescript < Formula
   desc "Clojure to JS compiler"
   homepage "https://github.com/clojure/clojurescript"
-  url "https://github.com/clojure/clojurescript/releases/download/r1.10.758/cljs.jar"
-  sha256 "dcc98e103d281d4eab21ca94fba11728e9f587c3aa09c8ffc3b96cff210adcce"
+  url "https://search.maven.org/remotecontent?filepath=org/clojure/clojurescript/1.10.844/clojurescript-1.10.844.jar"
```
Comment (chenrui333) [inline Formula/clojurescript.rb:4]: the cljs.jar is not attached in the latest github release.

**Review by carlocab — COMMENTED** [review]
This needs updating to account for the new source artefact:

https://github.com/Homebrew/homebrew-core/blob/7b043cf74826c5e24348502d81153aa54e19832c/Formula/clojurescript.rb#L20-L23

`livecheck` block also needs updating. Will probably work as is with just `url :head`, but maybe we want to change it to work with the stable URL.

**Comment by chenrui333** [comment]
Yeah, I am also checking with the upstream whether I should use the maven jar one. :)

**Comment by chenrui333** [comment]
Since they have updated the latest release with the jar, I have reverted to use the github release jar.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**